src/monitors/rmt.ts

```diff
@@ -74,9 +75,9 @@ export class TeamRatingMonitor extends BaseMonitor {
 		if (!res.length) {
 			return false; // No results
 		} else {
-			res = res.filter(r => {
-				const user = this.getUser(r.userid);
-				if (!user || user.presence.status === 'offline') return false;
+			res = res.filter(async (r) => {
```

Comment:
I don't think `Array#filter` will [handle the async callback properly](https://stackoverflow.com/questions/47095019/how-to-use-array-prototype-filter-with-async); it's probably most efficient to just use a `for` loop here and splice out the offline users.